docs/erp_fhirversion_changes.adoc

```diff
@@ -50,7 +50,9 @@ Um die Umbenennung von Profilen oder deren Umlagerung in neue Profile festzuhalt
 * xref:./resources/transitionfiles/transitionfile_de.gkvsv.eRezeptAbrechnungsdaten_v1.2.0_to_v1.3.0.json[transitionfile_de.gkvsv.eRezeptAbrechnungsdaten_v1.2.0_to_v1.3.0.json]
 
 == Generierte Vergleichs Ergebnisse
-Die Ergebnisse der Abgleiche können hier eingesehen werden:
+NOTE: Die Abhängigkeiten der vorgestellten Komponenten können auf Simplifier unter dem Tab "Dependencies" eingesehen werden. Zum Beispiel können Sie die Abhängigkeiten des eRezept-Workflows unter https://simplifier.net/erezept-workflow/~dependencies einsehen. Für Änderungen innerhalb der Abhängigkeiten empfehlen wir Ihnen, die Releasenotes der Pakete zu analysieren. Beispielsweise finden Sie eine Liste der Releasenotes für das Paket "de.basisprofil.r4" in Version 1.3.2 unter https://simplifier.net/packages/de.basisprofil.r4/1.3.2.
```

Comment:
Vielleicht noch einen einleitenden Satz, der Beschreibt, dass wir diese Unterabhängigkeiten nicht in Gänze vergleichen?